app/assets/javascripts/admin/members.js

```diff
@@ -156,4 +156,85 @@ $(document).on("ready page:load turbolinks:load", function () {
   });
 
   destroy(null);
+
+  $("#cards button").on("click", button_action);
+
+  function button_action() {
```

Comment:
This function also get's called in admin/checkout when activating / removing cards not really sure how but it happens